merchants/verkkokauppa/helpers.py

```diff
@@ -23,7 +23,7 @@
 def parse_datetime(string: Optional[str]) -> Optional[datetime]:
     if string is None:
         return None
-    return datetime.strptime(string, "%Y-%m-%dT%H:%M:%S.%f")
+    return datetime.strptime(string, "%Y-%m-%dT%H:%M:%S.%f")  # what is the timezone?
```

Comment:
I pondered how does the verkkokauppa work with timezones. Does it send the times without zone info and we expect them to be UTC or UTC+3 🕵️

Reply:
I asked this from Severi and he told me all times are UTC

Reply:
Oh and feel free to change to to explicitly read the datetime as UTC. I already approved the rest of the PR 😄